File: Source/JapaneseLanguageTools.Contracts/Models/Responses/GenerateCharacterExerciseBatchResponse.Model.cs
using System;

using JapaneseLanguageTools.Contracts.Models.Requests;

namespace JapaneseLanguageTools.Contracts.Models.Responses;

public class GenerateCharacterExerciseBatchResponseModel
{
    public required Guid CharacterExerciseBatchId { get; set; }

    public required GenerateCharacterExerciseBatchRequestModel Request { get; set; }
}

File: Source/JapaneseLanguageTools.ReactSpa/source/models/responses/GenerateCharacterExerciseBatchResponse.Model.ts
import GenerateCharacterExerciseBatchRequestModel from "@/models/requests/GenerateCharacterExerciseBatchRequest.Model";

export default interface GenerateCharacterExerciseBatchResponseModel {
  characterExerciseBatchId: string;

  request: GenerateCharacterExerciseBatchRequestModel;
}
